vllm/v1/core/sched/scheduler.py

```diff
@@ -33,6 +34,7 @@ class Scheduler(SchedulerInterface):
 
     def __init__(
         self,
+        vllm_config: VllmConfig,
```

Comment:
Its very confusing to have a class that gets `VllmConfig` and all these configs.

We shoudl either * just * pass vllm_config or pass `kv_transfer_config` directly